frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/PipelineRunDrawerRightContent.tsx

```diff
@@ -46,7 +51,10 @@ const PipelineRunDrawerRightContent: React.FC<PipelineRunDrawerRightContentProps
           <DrawerCloseButton onClick={onClose} />
         </DrawerActions>
       </DrawerHead>
-      <DrawerPanelBody style={{ display: 'flex', flexDirection: 'column' }}>
+      <DrawerPanelBody
+        className="pipeline-run__drawer-panel-body pf-v5-u-pr-sm"
+        // pf-v5-u-pr-sm
```

Comment:
@ppadti A small nit: I don't think this comment line is helpful. Either insert a meaningful comment or remove this line altogether.

Reply:
nvm, will be removing this comment while rebasing my PR. Thanks.